corehq/apps/linked_domain/tests/test_update_toggles.py

```diff
@@ -0,0 +1,43 @@
+from unittest import mock
+
+from corehq.apps.linked_domain.local_accessors import get_enabled_toggles
+from corehq.apps.linked_domain.tests.test_linked_apps import BaseLinkedAppsTest
+from corehq.apps.linked_domain.updates import update_toggles
+from toggle.shortcuts import set_toggle
+from corehq.toggles import NAMESPACE_DOMAIN, StaticToggle, TAG_INTERNAL
+
+TEST_TOGGLE = StaticToggle(
+    'test_toggle',
+    'This is for tests',
+    TAG_INTERNAL,
+    [NAMESPACE_DOMAIN],
+)
+
+
+@mock.patch('corehq.toggles.all_toggles_by_name', mock.Mock(return_value={'TEST_TOGGLE': TEST_TOGGLE}))
+class TestUpdateToggles(BaseLinkedAppsTest):
+
+    def tearDown(self):
+        super(TestUpdateToggles, self).tearDown()
+        set_toggle('test_toggle', self.domain_link.master_domain, False, NAMESPACE_DOMAIN)
+        set_toggle('test_toggle', self.domain_link.linked_domain, False, NAMESPACE_DOMAIN)
+
+    def test_get_enabled_toggles(self):
+        set_toggle('test_toggle', self.domain_link.master_domain, True, NAMESPACE_DOMAIN)
+        set_toggle('report_builder', self.domain_link.master_domain, True, NAMESPACE_DOMAIN)
```

Comment:
Huh I forgot to remove this line. Curious that this test passes. Looking into it now.

Reply:
Oh I'm mocking the method nevermind. Exhibit A of why mocking things isn't ideal. Couldn't help but hear Dumbledore saying "Actually if I think about it, it doesn't seem curious at all."